docs/source/support/release-notes.rst

```diff
@@ -6,6 +6,110 @@ Release notes
 
 This page contains release notes for the SDK.
 
+.. _release-0-13-30:
+
+0.13.30 - 2019-10-15
+--------------------
+
+DAML Standard Library
+~~~~~~~~~~~~~~~~~~~~~
+
+- Add ``DA.Action.State`` module containing a ``State`` action that
+  can be used for computations that modify a state variable.
+
+DAML Compiler
+~~~~~~~~~~~~~
+
+- Fixed the location of interface files when the
+  ``source`` field in ``daml.yaml`` points to a file. This is mainly
+  important for when you want to use the created ``.dar`` in the
+  ``dependencies`` field of another package.
+  See `issue #3135 <https://github.com/digital-asset/daml/issues/3135>`_.
+
+DAML-LF
+~~~~~~~
+
+- **Breaking** archive proto package renamed from `com.digitalasset.daml_lf` to `com.digitalasset.daml_lf_dev`
```

Comment:
I wonder if we should tone this down.
```
Rename DAML-LF Archive protobuf package from `com.digitalasset.daml_lf` to `com.digitalasset.daml_lf_dev`. This will only affect you do not use the DAML-LF Archive reader provided with the SDK but a custom one based on code generation by protoc.

Reply:
:+1: